src/features/common/Layout/ErrorHandlingContext.tsx

```diff
@@ -6,60 +6,37 @@ import React, {
   Dispatch,
   FC,
 } from 'react';
-import { UserPropsContext } from './UserPropsContext';
+import { SerializedError } from '@planet-sdk/common';
 
 type SetState<T> = Dispatch<SetStateAction<T>>;
 
-export interface ErrorInterface {
-  type?: 'error' | 'warning' | 'info';
-  message: string;
-  redirect?: string;
-  code?: number;
-}
-
 interface ErrorHandlingContextInterface {
-  error: ErrorInterface | null;
-  setError: SetState<ErrorInterface | null>;
-  handleError: (error: ErrorInterface) => void;
+  errors: SerializedError[] | null;
+  setErrors: SetState<SerializedError[] | null>;
+  redirect: (url: string) => void;
```

Comment:
To avoid importing `useRouter` and destructuring `push` method on every single page where we need it.
It is cleaner to just call it from the context.